### PR TITLE
Delete the third party javascript folder when pip install -e

### LINKED ISSUE #987: rollup script should empty third-party directory first
Currently, the rollup script (which runs as part of `pip install .`) onlly adds and overwrites to the `src/fontra/client/third-party/`. But anything that was there before that it doesn't write stays. This makes it a little harder to test the removal of things (such as recently with lit). The rollup script should empty (or delete) the third-party folder before it writes the new contents.

### PR DESCRIPTION
**Pull request by fatih-erikli**
Fixes #987.

### CODE REVIEW

**Comment by justvanrossum** [comment]
Sorry, this is the wrong approach (too complicated).

Please do `shutil.rmtree()` in https://github.com/googlefonts/fontra/blob/main/scripts/rollup_build_hook.py, using `pathlib.Path(__file__).resolve().parent` etc. to find the right directory.

File: scripts/rollup_build_hook.py [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+import shutil
 import subprocess
 
 from hatchling.builders.hooks.plugin.interface import BuildHookInterface
 
 
 class RollupBuildHook(BuildHookInterface):
     def initialize(self, version, build_data):
+        path = (
+            pathlib.Path(__file__).parent
```
Comment (justvanrossum) [inline scripts/rollup_build_hook.py:11]: Adding ".." is the same as adding another `.parent`. Please do `.parent` instead.
Also, the resolve should come first, not last:
`pathlib.Path(__file__).resolve().parent.parent ...`

**Review by justvanrossum — COMMENTED** [review]
This looks mostly good now.

Btw I closed the PR to indicate that I wanted you to start a new one. We don't need all those earlier commits, so I will have to do a squash commit once it's done.

**Comment by fatih-erikli** [comment]
I deleted earlier commits, no need to squash now.

File: scripts/rollup_build_hook.py [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
     def initialize(self, version, build_data):
+        path = (
+            pathlib.Path(__file__).resolve().parent.parent
+            / "src"
+            / "fontra"
+            / "client"
+            / "third-party"
+        )
+        if path.exists():
+            shutil.rmtree(path.resolve())
```
Comment (justvanrossum) [inline scripts/rollup_build_hook.py:18]: This .resolve() is now redundant.

Comment (fatih-erikli) [reply scripts/rollup_build_hook.py]: ok

**Pull request merged**